addons/wardrobe/functions/fnc_nextVariantGet.sqf

```diff
@@ -0,0 +1,58 @@
+#include "../script_component.hpp"
+/*
+ * Author: OverlordZorn
+ * Context Menu. Return config of the "next" variant - if multiple variants are defined, it will choose randomly.
+ * Maintains a history of the current and chosen variants as long as the inventory is open.
+ * Already used variants from the hstory will be removed from the availabe options.
+ * Once all options have been exhausted, the history will be reset.
+ *
+ * Arguments:
+ * 0: Current variant <CONFIG>
+ *
+ * Return Value:
+ * Desired variant <CONFIG>
+ *
+ * Example:
+ * _cfgCurrent call ace_wardrobe_fnc_nextVariantGet
+ *
+ * Public: No
+ */
+
+params ["_cfgCurrent"];
+
+private _cfgModifiableTo = [_cfgCurrent, false] call FUNC(getItemsModifiableTo);
+
+if (_cfgModifiableTo isEqualTo []) exitWith {false};
```

Comment:
This function shouldn't be returning a non-config unless it absolutely needs to. If a bool is required, update the header, otherwise apply suggestion.
```suggestion
if (_cfgModifiableTo isEqualTo []) exitWith {configNull};
```